### DISCUSSION TITLE
Can not return translation in nested json

### DISCUSSION THREAD

**Discussion by NaeemSohrabi** [spatie/laravel-translatable#311] [Q&A]
Hi 
i have a nested json like this in column home:
{
    "tags": {
        "en": [
            "one",
            "tow"
        ],
        "fa": [
            "اول"
        ]
    },
    "description": {
        "en": "<p>lorem ipsum</p>",
        "fa": "<p>لورم ایپسوم</p>"
    }
}
that the languages are in second level
and in my model , translatable is like this:
`public $translatable = ['home.*'];`
but it returns both fa and en languages.
in other columns that is like this:
{
    "en": "lorem",
    "fa": "الورم"
}
and in my model , translatable is like this:
`public $translatable = ['title'];`
translate works fine. , but in that case does not.
How should i use translate for that nested json????
please help
Thanks

**Comment by NaeemSohrabi** [comment]
Does not have any answer??

**Reply by arnarstef** [reply]
I would also like an answer here as I'm working with nested jsons. Is there a way to do this?
